modules/configuring-hybrid-ovnkubernetes.adoc

```diff
@@ -73,8 +73,13 @@ spec:
         hybridOverlayVXLANPort: 9898 <2>
 ----
 <1> Specify the CIDR configuration used for nodes on the additional overlay network. The `hybridClusterNetwork` CIDR cannot overlap with the `clusterNetwork` CIDR.
-<2> Specify a custom VXLAN port for the additional overlay network. This is required for running Windows nodes in a cluster installed on vSphere, and must not be configured for any other cloud provider. The custom port can be any open port excluding the default `4789` port. For more information on this requirement, see the Microsoft documentation on link:https://docs.microsoft.com/en-us/virtualization/windowscontainers/kubernetes/common-problems#pod-to-pod-connectivity-between-hosts-is-broken-on-my-kubernetes-cluster-running-on-vsphere[Pod-to-pod connectivity between hosts is broken]. 
+<2> Specify a custom VXLAN port for the additional overlay network. This is required for running Windows nodes in a cluster installed on vSphere, and must not be configured for any other cloud provider. The custom port can be any open port excluding the default `4789` port. For more information on this requirement, see the Microsoft documentation on link:https://docs.microsoft.com/en-us/virtualization/windowscontainers/kubernetes/common-problems#pod-to-pod-connectivity-between-hosts-is-broken-on-my-kubernetes-cluster-running-on-vsphere[Pod-to-pod connectivity between hosts is broken].
 --
++
+[NOTE]
+====
+Windows Server Long-Term Servicing Channel (LTSC): Windows Server 2019 is not supported on clusters with a custom hybridOverlayVXLANPort since this Windows server version does not support selecting a custom VXLAN port.
```

Comment:
Suggestion:

```
... clusters with a custom `hybridOverlayVXLANPort` value because Windows Server 2019 does not support ...
```

Reply:
You can't just say `Windows Server 2019`. You would have to say the full `Windows Server Long-Term Servicing Channel (LTSC): Windows Server 2019` or `Windows Server 1809`. Yes Windows versions are confusing  😄

Reply:
ACK. Sorry, didn't realize that's how the colon was being used there. :]